scenes/player/Player2d.gd

```diff
@@ -30,5 +30,13 @@ func get_input():
 
 func _physics_process(_delta):
 	get_input()
-	
 	move_and_slide()
+
+
+
+
+func _on_area_2d_area_entered(area):
+	print(health)
+	health = health - 1
+	if health == 0:
```

Comment:
What happens if the player gets hit by more than one damage and they are below zero?